src/core/operations/RegularExpression.mjs

```diff
@@ -241,8 +242,16 @@ function regexHighlight (input, regex, displayTotal) {
         // Add up to match
         output += Utils.escapeHtml(input.slice(i, m.index));
 
+        title = "Offset: " + m.index + "&#10;";
+        if (m.length > 1) {
+            title += "Groups:&#10;";
+            for (let n = 1; n < m.length; ++n) {
+                title += n + ": " + m[n] + "&#10;";
```

Comment:
Using template strings here would be more clear. 
Is there a reason you used `&#10;` instead of `\n`?